frontend_egui/src/app.rs

```diff
@@ -389,7 +393,23 @@ impl SnowGui {
                     ui.close_menu();
                 }
                 ui.separator();
+                ui.menu_button("Recent workspaces", |ui| {
+                    for (i, path, display_name) in self.settings.get_recent_workspaces_for_display() {
+                        if ui
```

Comment:
Formatting check fails here. You can run `cargo fmt` to auto format all your changes :)